src/GlobalStyles.js

```diff
@@ -11,6 +11,15 @@ export const GlobalStyles = createGlobalStyle`
     box-sizing: inherit;
   }
 
+  body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
```

Comment:
I don't think that margin is needed 👌

Reply:
I set zero margin for the entire body of the page because it can be useful in many cases, for example: you can remove the default margin, which allows you to better control the layout of the page as well as ensure consistency in appearance and layout between different browsers.

Reply:
The thing is our normalize.css sets the margin: 0; by default, so its kinda unnecessary from my point of view 👌

Reply:
I remove well